docs/FAQ/FAQ.md

```diff
@@ -6,63 +6,50 @@ Here are some commonly-asked questions and their answers. Another source of info
-
-    Upgrade to VIC 4.1.1 or later. This problem does not exist in later versions of the model.
+    2.  Check your model time step or snow step - does the problem disappear if you reduce the time step length?
+    3.  If your inputs are reasonable, consider setting `TFALLBACK` to TRUE in the *global parameter file*. Now, the simulation will continue using the previous time step's temperature value. The number of instances in which the previous step's T value was used will be reported at the end of each grid cell's simulation. In addition, several output variables are available to monitor when these instances occurred: `OUT_TFOL_FBFLAG`, `OUT_TCAN_FBFLAG`, `OUT_SURFT_FBFLAG`, `OUT_SOILT_FBFLAG`.
+    4.  If you use the `TFALLBACK` option, make sure to output the various temperatures simulated by the model (`OUT_TFOLIAGE`, `OUT_TCANOPY`, `OUT_SURFT`, `OUT_SOILT`) and check that these values look reasonable. In some cases, convergence failures occur long after the model has ventured into unphysical behavior, and using the previous time step's T value simply preserves these unphysical temperatures.
+    5.  Check if these behaviors are a [known issue under investigation](../Development/ModelDevelopment.md). If you do not see the issue listed, please create an issue [on Github](https://github.com/UW-Hydro/VIC/issues) and tell us about the issue.
 
-11.  **VIC 4.0.x sometimes gives large water balance errors when COMPUTE_TREELINE is TRUE - what can I do?**
+8.  **Why does actual evapotranspiration (ET) sometimes exceed potential evapotranspiration (PET)?**
 
-    Please ignore these water balance errors. They are accounting errors only, resulting from a bug in the water balance check, rather than any problems in any water balance terms. These errors only occur in the water balance of the first time step of the simulation. They only occur in grid cells that have elevation bands above the treeline. These errors do not occur if COMPUTE_TREELINE is FALSE. These errors do not occur in VIC 4.1.1 or later.
+    VIC's actual evapotranspiration is the sum of canopy evaporation, transpiration, soil evaporation, canopy snow sublimation, and ground snow sublimation, averaged over the grid cell.  VIC's `OUT_PET` (release 5.0 onwards; `OUT_PET_NATVEG` for releases 4.1-4.2) computes PET as the ET that the current landscape (with its current vegetation, architectural resistance, and LAI) would produce in the absence of limitations from soil moisture, vapor pressure deficit, temperature, or insolation.  But VIC's canopy evaporation and sublimation are computed using a canopy resistance of 0, which allows for much higher ET rates than when architectural resistance and LAI are taken into account.  Thus, when there is water stored in the canopy, actual ET can exceed `OUT_PET`.  This generally happens during a small fraction of the time of a simulation (immediately following rain events, for example).
```

Comment:
@tbohn - any comment here?  I just changed the formatting of the output variables.